api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -224,6 +224,16 @@ message GrpcJsonTranscoder {
 
   // If true, try to match the custom verb even if it is unregistered. By
   // default, only match when it is registered.
+  //
+  // According to the http template syntax(https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L226-L231),
```

Comment:
envoy comment is not using markdown syntax,  it is using something else, the link format is wrong